test/tests/01-package/run.sh

```diff
@@ -8,7 +8,7 @@ then
   major="${RESTY_IMAGE_TAG%%.*}"
   docker run -d --name user-validation-tests --rm -e KONG_DATABASE=off -v $PWD:/src registry.access.redhat.com/ubi${major}/ubi tail -f /dev/null
 else
-  docker run -d --name user-validation-tests --rm -e KONG_DATABASE=off -v $PWD:/src ${RESTY_IMAGE_BASE}:${RESTY_IMAGE_TAG} tail -f /dev/null
+  docker run -d --name user-validation-tests --rm -e KONG_DATABASE=off -v $PWD:/src --platform linux/amd64 ${RESTY_IMAGE_BASE}:${RESTY_IMAGE_TAG} tail -f /dev/null
```

Comment:
Does this mean we aren't tailoring the tests to their specific architectures?

Reply:
This is so when the host system is amd64 the build doesn't go
```
build amd64
build arm64
accidentally use arm64 for testing because it's what's laying around
```

Now that I type this out it's occurred to me this will fix the bug for me (amd64) but break for m2 (arm64) :(

Reply:
```suggestion
  docker rmi ${RESTY_IMAGE_BASE}:${RESTY_IMAGE_TAG} --force
  docker run -d --name user-validation-tests --rm -e KONG_DATABASE=off -v $PWD:/src ${RESTY_IMAGE_BASE}:${RESTY_IMAGE_TAG} tail -f /dev/null
```

I think will fix the situation right?